napari/view_layers.py

```diff
@@ -263,126 +263,108 @@ def imshow(
-        that is being expanded and then each colormap is applied to each
-        image.
-    gamma : list, float
-        Gamma correction for determining colormap linearity. Defaults to 1.
-        If a list then must be same length as the axis that is being
-        expanded as channels.
-    interpolation : str or list
-        Deprecated, to be removed in 0.6.0
-    interpolation2d : str or list
-        Interpolation mode used by vispy in 2D. Must be one of our supported
-        modes. If a list then must be same length as the axis that is being
-        expanded as channels.
-    interpolation3d : str or list
-        Interpolation mode used by vispy in 3D. Must be one of our supported
-        modes. If a list then must be same length as the axis that is being
-        expanded as channels.
-    rendering : str or list
-        Rendering mode used by vispy. Must be one of our supported
-        modes. If a list then must be same length as the axis that is being
-        expanded as channels.
-    depiction : str
-        Selects a preset volume depiction mode in vispy
-
-        * volume: images are rendered as 3D volumes.
-        * plane: images are rendered as 2D planes embedded in 3D.
-    iso_threshold : float or list
-        Threshold for isosurface. If a list then must be same length as the
-        axis that is being expanded as channels.
-    attenuation : float or list
-        Attenuation rate for attenuated maximum intensity projection. If a
-        list then must be same length as the axis that is being expanded as
-        channels.
-    name : str or list of str
-        Name of the layer.  If a list then must be same length as the axis
-        that is being expanded as channels.
-    metadata : dict or list of dict
-        Layer metadata. If a list then must be a list of dicts with the
-        same length as the axis that is being expanded as channels.
-    scale : tuple of float or list
-        Scale factors for the layer. If a list then must be a list of
-        tuples of float with the same length as the axis that is being
-        expanded as channels.
-    translate : tuple of float or list
-        Translation values for the layer. If a list then must be a list of
-        tuples of float with the same length as the axis that is being
-        expanded as channels.
-    rotate : float, 3-tuple of float, n-D array or list.
-        If a float convert into a 2D rotation matrix using that value as an
-        angle. If 3-tuple convert into a 3D rotation matrix, using a yaw,
-        pitch, roll convention. Otherwise assume an nD rotation. Angles are
-        assumed to be in degrees. They can be converted from radians with
-        np.degrees if needed. If a list then must have same length as
+        All parameters except data, rgb, and multiscale can be provided as
+        list of values. If a list is provided, it must be the same length as
```

Comment:
```suggestion
        All parameters except rgb, and multiscale can be provided as
        list of values. Data is a special paramter which can be provided as a 
        list of arrays, but in this case it is treated as multiscale and not 
        as individual layers. For the other parameters, if a list is provided, 
        it must be the same length as
```

Reply:
@melonora we are discussing this suggestion in the PR party. The problem here is that the extent of possibilities is very complex, and neither the existing simple comment nor your suggestion fully capture the complexity — but the simple version at least is less confusing to a new user.

So we think, let's merge this as-is, and we can open a new PR / issue to discuss the exact wording of the docstring.